com.obsproject.Studio.Plugin.Kinect.yaml

```diff
@@ -0,0 +1,58 @@
+id: com.obsproject.Studio.Plugin.Kinect
+branch: stable
+runtime: com.obsproject.Studio
+runtime-version: stable
+sdk: org.kde.Sdk//6.4
+build-extension: true
+separate-locales: false
+appstream-compose: false
+build-options:
+  prefix: /app/plugins/Kinect
+  prepend-pkg-config-path: /app/plugins/Kinect/lib/pkgconfig # Needed for libfreenect2
+cleanup:
+  - /lib/Kinect/*.a
+modules:
+  - shared-modules/libusb/libusb.json
+
+  - name: libfreenect
+    buildsystem: cmake-ninja
+    cleanup:
+      - /lib/pkgconfig
+      - /lib/fakenect
+      - /include
+      - /bin
+    sources:
+      - type: git
+        url: https://github.com/SirLynix/libfreenect.git
+        commit: 5e2cd8e18877fbe7ef4c863161e90873c06a9180
```

Comment:
Is there no specific tag that can be used here? Sadly using commit hashes is a bit fragile, there were incidents in the past where the repository force-pushed and removed the commit.

Reply:
it can happen with tags too. or even deleted repositories. using a commit isn't a problem on the flathub side.

Reply:
The plugin author has some patches for libfreenect (kinect v1 support) that hasen't been upstreamed yet, and doesn't look like it will anytime soon (more on that on another comment). How about I use the latest upstream tag and add those patches in a file for flatpak builder to apply?

Reply:
Actually, it looks like they've fixed it a week ago! But I'd still have to make a patch file till they make a new release.